### PR TITLE
MMADetector spectrum

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds spectrum for MMA detectors.

### CODE REVIEW

File: skyportal/models/mmadetector.py [hunk 1]
```diff
@@ -74,6 +78,186 @@ class MMADetector(Base):
...
+        doc="MMADetectorSpectra of the object.",
+    )
+
 
 GcnEventMMADetector = join_model("gcnevents_mmadetectors", GcnEvent, MMADetector)
 GcnEventMMADetector.__doc__ = "Join table mapping GcnEvents to MMADetectors."
+
+
+class MMADetectorSpectrum(Base):
+    """Wavelength-dependent measurement of the sensitivity the detector."""
```
Comment (guynir42) [inline skyportal/models/mmadetector.py:97]: I would argue that Frequency dependent is a much better way to look at it.

File: skyportal/tests/data/aligo_O4high.txt [hunk 2]
```diff
@@ -0,0 +1,2736 @@
+1.021659e+01 6.662774e-21
```
Comment (guynir42) [inline skyportal/tests/data/aligo_O4high.txt:1]: Can we name this something more informative (the words "noise" and "spectrum" should be up there with "LIGO").

**Review by guynir42 — APPROVED** [review]
I have minor comments. 
Please update your migration hash, as it may have changed since the tests were last run.

File: skyportal/tests/data/aligo_O4high.txt (hunk 2 above)
Comment (mcoughlin) [reply skyportal/tests/data/aligo_O4high.txt]: Yeah added!

File: skyportal/models/mmadetector.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/models/mmadetector.py]: OOps.

**Pull request merged**